eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/RatifySpec.hs

```diff
@@ -487,6 +487,58 @@ votingSpec =
           passNEpochs 2
           -- The same vote should now successfully ratify the proposal
           getLastEnactedCommittee `shouldReturn` SJust (GovPurposeId addCCGaid)
+        it "Rewards contribute to active voting stake even in the absence of StakeDistr" $ do
+          -- Only modify the applicable thresholds
+          modifyPParams $ \pp ->
+            pp
+              & ppDRepVotingThresholdsL
+                .~ def
+                  { dvtCommitteeNormal = 51 %! 100
+                  , dvtCommitteeNoConfidence = 51 %! 100
+                  }
+              & ppCommitteeMaxTermLengthL .~ EpochInterval 20
+          -- Setup DRep delegation #1
+          (drepKH1, stakingKH1) <- setupDRepWithoutStake
+          -- Add rewards to delegation #1
+          modifyNES $
+            nesEsL . epochStateUMapL
+              %~ UM.adjust
+                (\(UM.RDPair r d) -> UM.RDPair (r <> UM.CompactCoin 1_000_000) d)
+                (KeyHashObj stakingKH1)
+                . UM.RewDepUView
```

Comment:
We should not be modifying the state manually in the imp test framework.

What is the way this can happen in production? Easy: submit a proposal and wait for it to expire. This will put the refund into the reward account.
Alternatively register and retire a stake pool will achieve the same: refund placed into the reward account
```suggestion
          -- Add rewards to delegation #1
```